src/Graphics/EasyImage.purs

```diff
@@ -0,0 +1,10 @@
+module Graphics.EasyImage where
+
+import Prelude (Unit())
+import Control.Monad.Aff (Aff())
+
+foreign import data EASY_IMAGE :: !
+
+foreign import cropInPlace :: forall e.
+                              Int -> Int -> Int -> Int ->
```

Comment:
This is just one function from all `easyimage` API. I think it will be replaced as this lib become more mature